.eslintignore

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: 2023 spdx contributors
```

Comment:
```suggestion
# SPDX-FileCopyrightText: 2023 SPDX contributors
```

the name `SPDX` is expected to be written in all-caps. Sorry for linking a bad example.

Reply:
Replaced it as suggested.